cmd/gen-go-sample/gapic.go

```diff
@@ -49,6 +49,7 @@ type SampleParameter struct {
 type SampleAttribute struct {
 	Parameter          string
 	SampleArgumentName string `yaml:"sample_argument_name"`
+	ReadFile bool `yaml:"read_file"`
```

Comment:
Run `go fmt`. I think the type should be aligned with the previous line. This probably means we should add a format check to CI as well.